src/path_info.cpp

```diff
@@ -158,6 +158,12 @@ void PATH_INFO::set_standard_filenames(void)
     update_pathname("obsolete-mods", FILENAMES["moddir"] + "obsolete-mods.json");
     update_pathname("defaultsounddir", FILENAMES["datadir"] + "sound");
 
+    // User's data directories
+    update_pathname("user_datadir", FILENAMES["user_dir"] + "data/");
+    update_pathname("user_gfxdir", FILENAMES["user_datadir"] + "gfx/");
```

Comment:
Ah, I see. For the case you describe, you have to define neither USE_HOME_DIR nor USE_XDG_DIR.

gfx is the only one that works properly actually, because the paths are different. sound and mod paths are the equivalent; however the strings are different, causing the string comparison to pass when it shouldn't. I knew those naive string comparisons would fail for path comparisons. :)

I'll see if I can figure out a solution to compare filesystem paths w/o bringing in much code/dependencies.

Just curious, why even have the option to use the current directory for user_dir? The gentoo ebuild actually had a bug where it could fail to define one of those, causing save games, etc to spill out into current directory (https://github.com/leycec/raiagent/pull/44). Is it just to keep dev save games separate?